contracts/LinearPremiumPriceOracle.sol

```diff
@@ -0,0 +1,56 @@
+pragma solidity >=0.5.0;
+
+import "./SafeMath.sol";
+import "./StablePriceOracle.sol";
+
+contract LinearPremiumPriceOracle is StablePriceOracle {
```

Comment:
Don't you need to define `INTERFACE_META_ID` ?

`bytes4(keccak256("timeUntilPremium(uint,uint)"));` is `0x5e75f6a9` I think

Reply:
Good catch.